config/fortify.php

```diff
@@ -50,6 +50,19 @@
 
     'email' => 'EmailAddress',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Lowercase Usernames
+    |--------------------------------------------------------------------------
+    |
+    | This value defines whether usernames should be lowercased before saving
+    | them in the database, as some database system string fields are case
+    | sensitive. You may disable this for your application if necessary.
+    |
+    */
+
+    'lowercase_usernames' => true,
```

Comment:
This seems like a significant change. What about all the existing users?

Reply:
good point, this will be changed. fortify is not yet used for user registration, will follow in another PR where public scripts will be replaced.

i'm still thinking about lowercasing all the usernames and moving the current usernames to the display name column. this makes the integration of fortify much easier and follows the v2 prototype's approach.
queries will have to be checked for this change, too, but as we're using a case insensitive collation we might be good without any additional changes.

Reply:
nothing that has to be decided just yet imo. that's just the default